Sources/MapboxNavigation/NavigationMapView+RoadNameLabeling.swift

```diff
@@ -23,7 +26,7 @@ extension NavigationMapView {
             do {
                 try mapView.mapboxMap.style.addSource(streetsSource, id: sourceIdentifier)
             } catch {
-                NSLog("Failed to add \(sourceIdentifier) with error: \(error.localizedDescription).")
+                os_log("Failed to add %s with error: %s.", log: logger, type: .error, sourceIdentifier as CVarArg, error.localizedDescription)
```

Comment:
Why do we need casting to `CVarArg` here?

Reply:
Cast was needed since os_log uses `StaticString`